config/workers.coffee

```diff
@@ -318,7 +318,7 @@ module.exports = (KONFIG, options, credentials) ->
         ]
 
     userproxies         :
-      group             : 'proxy'
+      group             : if KONFIG.environment is 'default' then 'default' else 'proxy'
```

Comment:
you can add default into https://github.com/koding/koding/blob/master/deployment/grouptoenvmapping.coffee#L1 instead of this change